tests/test_extruct.py

```diff
@@ -17,9 +16,10 @@ def test_all(self):
         body = get_testdata('songkick', 'elysianfields.html')
         expected = json.loads(get_testdata('songkick', 'elysianfields.json').decode('UTF-8'))
         data = extruct.extract(body, base_url='http://www.songkick.com/artists/236156-elysian-fields')
-        # See test_rdfa_not_preserving_order()
-        del data['rdfa'][0]['http://ogp.me/ns#image']
-        del expected['rdfa'][0]['http://ogp.me/ns#image']
+        for rdf in data['rdfa']:
```

Comment:
Add a comment with the explanation about why you are sorting elements for rdfa. You can maybe add the link to the bug: See https://github.com/scrapinghub/extruct/issues/116

Reply:
Ok, it's a good idea 👍